modules/websocket-client/package.json

```diff
@@ -1,30 +1,34 @@
 {
-  "name": "phenyl-websocket-client",
-  "version": "0.2.9",
+  "name": "@phenyl/websocket-client",
+  "version": "1.0.0",
   "description": "",
+  "repository": "phenyl-js/phenyl",
   "main": "./dist/index.js",
-  "jsnext:main": "./jsnext.js",
+  "types": "./dist/index.d.js",
+  "scripts": {
+    "build": "upbin tsc --declaration"
+  },
   "files": [
     "src",
     "dist",
```

Comment:
`src`, `dist`, `package.json` can be removed from `files`. instead, please add `lib`